### PR TITLE
handling expired password

### PR DESCRIPTION
**Pull request by xpt1x**
I check with response, it shows object moved ( a 302 ) 
Dont know if this is the best way or the most accurate but i guess after login, they only redirect when pass expires

### CODE REVIEW

File: uims_api/session.py [hunk 1]
```diff
@@ -50,9 +50,10 @@ def _login(self):
                                  cookies=response.cookies,
                                  allow_redirects=False)
 
-        incorrect_credentials = response.status_code == 200
-        if incorrect_credentials:
-            raise IncorrectCredentialsError("Make sure UID and Password are correct.")
+        if response.status_code == 200:
+            raise IncorrectCredentialsError('Make sure UID and Password are correct')
+        elif response.status_code == 302:
+            raise PasswordExpiredError('You Password has expired! Please update it first')
```
Comment (ritiek) [inline uims_api/session.py]: Your*

Comment (xpt1x) [reply uims_api/session.py]: Aaah :smiling_imp:

Comment (xpt1x) [reply uims_api/session.py]: Done :heavy_check_mark:

**Review by ritiek — APPROVED** [review]
Looks good, thanks!

**Pull request merged**